modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -5581,8 +5582,14 @@ public ObjectActionModel newObjectActionModel(
 				data, "${fragmentEntryLinkId}",
 				String.valueOf(
 					fragmentEntryLinkModel.getFragmentEntryLinkId()));
+
+			segmentsExperienceId =
+				fragmentEntryLinkModel.getSegmentsExperienceId();
```

Comment:
@slnn 
> all the FragmentEntryLinkModels are using the same segmentsExperienceId so just use the last one's segmentsExperienceId

If this is the case, you can just `get(0)` from the `fragmentEntryLinkModels` list and set the one's segments experience id to the model. No need to repeatedly assign the same value to an variable.

Reply:
Done